cosim/black-parrot-example/ps.cpp

```diff
@@ -12,13 +12,19 @@
 #include <queue>
 #include <unistd.h>
 #include <bitset>
+#include <cstdint>
+#include <iostream>
+#ifdef FPGA
+#include <fstream>
+#include <thread>
+#endif
 
 #include "bp_zynq_pl.h"
 #include "bsg_printing.h"
 #include "bsg_argparse.h"
 
 #define FREE_DRAM 0
-#define DRAM_ALLOCATE_SIZE 120 * 1024 * 1024
+#define DRAM_ALLOCATE_SIZE 241 * 1024 * 1024
```

Comment:
Does this amount exist on Z2? Should this be passed in by Makefile?

Reply:
We have 256MB memory on both boards.

Reply:
Is 15 MB enough for the PS? seems small